docs/Dockerfile

```diff
@@ -5,7 +5,7 @@ MAINTAINER Mu Li <muli@cs.cmu.edu>
 # First, build MXNet binaries (ref mxnet/docker/cpu/Dockerfile)
 #
 
-RUN apt-get update && apt-get install -y build-essential git libopenblas-dev libopencv-dev
+RUN apt-get update && apt-get install -y build-essential git libopenblas-dev liblapack-dev libopencv-dev
```

Comment:
Another thing is that we'd need to patch another one or two small things in the Docker test run file as the Ubuntu 16.04 LTS Docker Image is much more compact than 14.04 and has removed packages as `sudo`.